geonode_mapstore_client/client/js/epics/gnsave.js

```diff
@@ -67,17 +67,16 @@ const SaveAPI = {
         const description = metadata.description;
         const thumbnail = metadata.thumbnail;
         const body = {
-            name,
             data,
```

Comment:
@giohappy @mattiagiupponi Just noticed that for the `api/v2/geostories` we include the data object as a string see [here](https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/client/js/epics/gnsave.js#L114) while the `api/v2/maps` accepts an object not stringified. Could we make the data property consistent between the requests of the v2 api? (I would prefer a not stringified object)

Reply:
> 
> 
> @giohappy @mattiagiupponi Just noticed that for the `api/v2/geostories` we include the data object as a string see [here](https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/client/js/epics/gnsave.js#L114) while the `api/v2/maps` accepts an object not stringified. Could we make the data property consistent between the requests of the v2 api? (I would prefer a not stringified object)

Yes, it should be better to maintain consistency between the apis. I leave the decition about stringy or not to @giohappy

Reply:
I'm fine with a not stringified version. Please open an issue to change the behaviour for the geostories API